content/applications/sales/point_of_sale/restaurant/tips.rst

```diff
@@ -1,31 +1,88 @@
-===================================
-Integrate a tip option into payment
-===================================
+====
+Tips
+====
 
-As it is customary to tip in many countries all over the world, it is
-important to have the option in your PoS interface.
+Tipping is customary in multiple countries. Point of Sale allows tipping in :doc:`shops
+<../overview/getting_started>`, :doc:`bars <../restaurant>`, or :doc:`restaurants <../restaurant>`.
 
-Configure Tipping
-=================
+.. _configuration:
 
-To activate the *Tips* feature, go to :menuselection:`Point of Sale
---> Configuration --> Point of sale` and select your PoS.
+Configuration
+=============
 
-Under the Bills & Receipts category, you will find *Tips*. Select it
-and create a *Tip Product* such as *Tips* in this case.
+To allow tipping in your POS, activate the :guilabel:`Tips` feature in :menuselection:`Point of Sale
+--> Configuration --> Settings`. At the top of the page, select the POS in which you wish to allow
+**tipping**, scroll down to the :guilabel:`Payment` section and check :guilabel:`Tips`. Once
+enabled, add a :guilabel:`Tip Product` in the corresponding field, and save. The designated product
+will be used as a reference on customers' receipts.
 
-.. image:: tips/tips01.png
-    :align: center
+.. image:: tips/tips-setup.png
+   :align: center
+   :alt: enable tips in a POS
 
-Add Tips to the bill
-====================
+.. _tip-product:
 
-Once on the payment interface, you now have a new *Tip* button
+Tip products
+------------
 
-.. image:: tips/tips02.png
-    :align: center
+**Tip products** can be created on the spot. To do so, enter a product's name in the :ref:`Tip
+Product <configuration>` field and click :guilabel:`Create` or press **enter**. The product is
+automatically configured to be used as a tip at the payment screen.
 
-Add the tip your customer wants to leave and process to the payment.
+However, if you wish to be able to select the tip product in a POS session, you must activate the
+**Available in POS** setting. To do so, click :guilabel:`Create and edit...` instead of
+:guilabel:`Create` after you enter the product's name to open the product configuration form. Then,
+tick the :guilabel:`Available in POS` checkbox in the :guilabel:`Sales` tab and click
+:guilabel:`Save & Close`.
```

Comment:
I suggest you directly explain where to go, rather than saying "click create and edit instead of create after you...."

--> To do so, go to the Sales tab of the tip product and tick....